src/generate_reducer.ts

```diff
@@ -1,6 +1,6 @@
 import { ReduxAction } from "./interfaces";
 
-const NOOP = (s, a) => s;
+const NOOP = (s: any, a: ReduxAction<{}>) => s;
```

Comment:
Actually, move `NOOP` into the function definition after the current first line.